src/constants/enums.ts

```diff
@@ -3,7 +3,7 @@ export enum IChainType {
   avalanche = 'avalanche',
   binance = 'binance',
   binancesmartchain = 'binancesmartchain',
-  polygon = 'polygon',
+  polygon = 'matic',
```

Comment:
why?

Reply:
reverted

Reply:
because all libs is using 'matic', but we named 'polygon'